webapp/src/main/java/com/box/l10n/mojito/service/asset/AssetService.java

```diff
@@ -155,18 +157,15 @@
             pollableFutureTaskResult.setExpectedSubTaskNumberOverride(1);
         }
 
-        //TODO pass the owner information, merge Xiaye's PR
-        String createdByFromXiaye = null;
+        User branchCreatedByUser = null;
 
-        User branchCreatedBy = null;
-
-        if (createdByFromXiaye != null) {
-            branchCreatedBy =  userService.getOrCreatePartialBasicUser(createdByFromXiaye);
+        if (branchCreatedByUsername != null) {
```

Comment:
that could go in a function

Reply:
yeah. make code clear. make change to following statement

User branchCreatedByUser = branchCreatedByUsername != null ? userService.getOrCreatePartialBasicUser(branchCreatedByUsername : auditorAware.getCurrentAuditor();